code/game/objects/items/weapons/handcuffs.dm

```diff
@@ -145,6 +145,20 @@
 		else
 			user << "<span class='warning'>You need one rod to make a wired rod!</span>"
 			return
+	else if(istype(I, /obj/item/stack/sheet/metal))
+		var/obj/item/weapon/restraints/legcuffs/bola/S = new /obj/item/weapon/restraints/legcuffs/bola

```

Comment:
You make a good point. If you do this but fail the do-after, I guess the bolas would get stuck in there. How do I fix that?